projects/ontimize-web-ngx/src/lib/services/ontimize/ontimize-export.service.ts

```diff
@@ -4,12 +4,12 @@ import { Router } from '@angular/router';
 import { Observable } from 'rxjs';
 import { share } from 'rxjs/operators';
 
-import { AppConfig } from '../config/app-config';
-import { IExportService } from '../interfaces';
-import { Config } from '../types/config.type';
-import { Codes } from '../util/codes';
-import { ServiceUtils } from '../util/service.utils';
-import { LoginStorageService } from './login-storage.service';
+import { AppConfig } from '../../config/app-config';
+import { IExportService } from '../../interfaces/export-service.interface';
+import { Config } from '../../types/config.type';
+import { Codes } from '../../util/codes';
+import { ServiceUtils } from '../../util/service.utils';
+import { LoginStorageService } from '../login-storage.service';
 
 @Injectable({
```

Comment:
Remove `providedIn: 'root'`. This service is provided at component level in table export dialog